spec/ca_testing/patches/logger_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+RSpec.describe CaTesting::Patches::Logger do
+  it_behaves_like "a patch" do
+    subject { described_class.new(CaTesting.logger) }
```

Comment:
HI, did you try this:

    let(:logger) { double("logger", level: :UNKNOWN), }
    let(:ca_testing) { double("CaTesting", logger: logger)}
    subject { described_class.new(ca_testing.logger) }